packages/fxa-payments-server/src/components/Survey/README.md

```diff
@@ -0,0 +1,30 @@
+### Example Usage
+
+```javascript
+
+  import Survey, { CreateHandleIframeTask } from '@fxa-components/Survey';
+
+  const [showSurvey, setShowSurvey] = useState(true);
+  const [surveyComplete, setSurveyComplete] = useState(false);
+
+  const handleIframeTask = CreateHandleIframeTask(() => {
+    setSurveyComplete(true);
+    setTimeout(() => {
+      setShowSurvey(false);
+    }, 300);
+  });
+
+  useEffect(() => {
+    window.addEventListener('message', handleIframeTask);
```

Comment:
I think a comment here around SurveyGizmo emitting this could be helpful, I wasn't sure why we needed this until scrolling further down and seeing `new MessageEvent` in the test.